tests/bgp/bgp_helpers.py

```diff
@@ -781,3 +786,62 @@ def check_bgp_neighbor(duthost):
         wait_until(300, 10, 0, duthost.check_bgp_session_state, bgp_neighbors),
         "bgp sessions {} are not up".format(bgp_neighbors)
     )
+
+
+def is_tcpdump_running(duthost, cmd):
+    check_cmd = "ps u -C tcpdump | grep '%s'" % cmd
+    if cmd in duthost.shell(check_cmd)["stdout"]:
+        return True
+    return False
+
+
+@contextlib.contextmanager
+def capture_bgp_packages_to_file(duthost, iface, save_path, ns):
```

Comment:
Renamed the previous `log_bgp_updates` function and moved it to this helper file.